packer/README.md

```diff
@@ -1,10 +1,147 @@
+specific scenarios. The first is if you have a Hyper-V VM Switch defined in
+Hyper-V already you can add `-vmSwitch` along with the name of your Switch.
+If this parameter is not specified, then a Switch with the name
+`packer-hyperv-iso` will be created. The second optional parameter is for if you
+run the build and see the following error output
+
+```
+hyperv-iso: Download failed unexpected EOF
+hyperv-iso: error downloading ISO: [unexpected EOF]
+...
+Build 'hyperv-iso' errored after 10 minutes 26 seconds: error downloading ISO: [unexpected EOF]
+```
+
+If you see this, then the build is having a problem downloading the Windows
+Server 2019 iso file, so you should try to manually download the iso from [this
+url](https://software-download.microsoft.com/download/pr/17763.737.190906-2324.rs5_release_svc_refresh_SERVER_EVAL_x64FRE_en-us_1.iso),
+move the iso file to an easily accessible location, and then use the `-isoUrl`
+parameter in the build script to specifiy where the iso file is and the build
+script will use that file instead of trying to download it.
+
+
+# Packer Build
 
 | Folder | Description |
 | -------- | -------- |
 | `build/` | Cached artifacts and log files |
 | `dist/` | Built virtual machine |
 | `mnt/` | Used by packer to mount to the floppy drive |
+
+Below are examples showing how to pass these two parameters to the build script.
+
+```powershell
+#default way to run build for base image to generate the base image. Folders that are created during the build process are `./build` and `./dist`.
+PS> ./build-vm.ps1 -PackerFile .\win2019-eval-base.pkr.hcl -VariablesFile .\base-variables.json
+
+```powershell
+#default way to run build for starter-kit image to generate the starter-kit image. Folders that are created during the build process are `./build` and `./dist`.
+PS> ./build-vm.ps1 -PackerFile .\sea-starter-kit-win2019-eval.pkr.hcl -VariablesFile .\starter-kit-variables.json
+
+#build with vmSwitch parameter for base image
+PS> ./build-vm.ps1 -PackerFile .\win2019-eval-base.pkr.hcl -VariablesFile .\base-variables.json -VMSwitch existingVMSwitchName
+
+#build with vmSwitch parameter for starter-kit image
+PS> ./build-vm.ps1 -PackerFile .\sea-starter-kit-win2019-eval.pkr.hcl -VariablesFile .\starter-kit-variables.json -VMSwitch existingVMSwitchName
+
+```
+> :exclamation: IMPORTANT: Disconnect from any VPNs. This will cause issues with Hypervisor
+> connectivity.
+
+The build will download and install Windows Server 2019 Evaluation edition, with
+a license that is valid for 180 days. NuGet Package Management is installed,
+followed by Chocolatey for automated software package installs of the following
+software: Dot Net Framework 4.8, Dot Net Core 3.1 SDK, SQL Server 2019 Express,
+SQL Server Management Studio, Google Chrome, and any of their Chocolatey package
+dependencies (Windows update packages for Dot Net).
+
+Next, the build invokes the installation for the database, ODS Tech Suite, Data
```

Comment:
starter-kit image build uses the base image provided in build folder and invokes the installation of ODS / API , Admin App, starter kit sample data, starter kit sample extension, sample validation and reporting artifacts.

Reply:
done